_data/master/navbars/usage.yml

```diff
@@ -17,8 +17,14 @@ toc:
       path: /usage/calicoctl/configure/etcd
     - title: Kubernetes API datastore options
       path: /usage/calicoctl/configure/kdd
-- title: Securing Calico's Prometheus endpoints 
-  path: /usage/policy/secure-metrics
+- title: Securing Calico communications
+  section:
+  - title: Scheduling to well-known nodes
```

Comment:
I don't think this should be its own page - there's not enough content for it.

Reply:
It helps with merges downstream.